### DISCUSSION TITLE
Why don't I have annotation function?

### DISCUSSION THREAD

**Discussion by owenew** [sumatrapdfreader/sumatrapdf#3325] [Q&A]
![image](https://user-images.githubusercontent.com/37006925/222903651-fd1b2da4-dce3-49b3-9aa8-ecf2798b3090.png)
This option is not available.
And I can't use keyboard shortcuts.
version 3.4.6

**Comment by GitHubRulesOK** [comment]
Annotation is available for PDF only since 3.2
Annotations were Improved in 3.3 relese so well established by 3.4.x
![image](https://user-images.githubusercontent.com/12754913/222904989-d2926bd7-2bc9-4ba5-80e3-26f4ec0ea2c6.png)
![image](https://user-images.githubusercontent.com/12754913/222905526-fb5f3ba6-bd9c-4c9c-873b-3fe6a8e8ef61.png)

**Reply by owenew** [reply]
The version I downloaded is the latest version,but I cant find the annotation

**Reply by GitHubRulesOK** [reply]
fresh download from official https://www.sumatrapdfreader.org/download-free-pdf-viewer

![image](https://user-images.githubusercontent.com/12754913/222936816-0bb50082-e4de-4290-ba11-4d9c4d982778.png)

**Comment by chickenday** [comment]
So am I. With rightclick there is no annotation button, at the same time the shortcut "a" doesn't work. It's newest version 3.4.6-64bit, windows11. By the way, the portable version is the same. 
@GitHubRulesOK 
Using advanced setting also doesn't work.
![Snipaste_2023-03-21_11-24-28](https://user-images.githubusercontent.com/46754933/226510977-92874125-7dee-46e4-8f1b-b66711e4962f.jpg)

**Reply by GitHubRulesOK** [reply]
@chickenday 
I am confused again here i simply download from official 3.4.6 download  

64 or 32 should not matter which  but if you add key commands  your probably blocking the inbuilt keyboard shortcuts you would only add u if trying to force it to a different command (also only one key case not 2)

![image](https://user-images.githubusercontent.com/12754913/226515376-b11d0655-1f76-47e4-a909-efb13445199f.png)

**Comment by avorobey** [comment]
Are you sure you're viewing a PDF file and not some other format (EPUB, CBZ, FB2, DJVU...)? Annotations only work in PDF files.

**Reply by GitHubRulesOK** [reply]
@avorobey whilst you are 100% correct there may be older users (like me:-) who used the old sidecar highlight feature in text and images and ebook formats. thus may be confused that its now PDF only

**Reply by chickenday** [reply]
Oh, I see. What I want to annotate is the epub formation. Another similar I am using can do this. I didn't expect
that.
